src/libs/ReportUtils.js

```diff
@@ -455,6 +455,11 @@ function isAllowedToComment(report) {
         return true;
     }
 
+    // If user opens public chat room directly from deep link after logout
```

Comment:
```suggestion
    // If unauthenticated user opens public chat room using deeplink, they do not have policies available and they cannot comment
```

Reply:
Agree with this improvement. It's a bit complex else